i18n/en.json

```diff
@@ -3519,6 +3519,10 @@
   "user.settings.sidebar.title": "Sidebar Settings",
   "user.settings.sidebar.unreadSectionDesc": "Unread channels will be sorted at the top of the channel sidebar until read.",
   "user.settings.sidebar.unreadSectionTitle": "Group unread channels",
+  "user.settings.sidebar.channelSwitcherSectionTitle": "Channel Switcher",
+  "user.settings.sidebar.on": "On",
+  "user.settings.sidebar.off": "Off",
+  "user.settings.sidebar.channelSwitcherSectionDesc": "The channel switcher is shown at the bottom of the sidebar and is used to jump between channels quickly. It can also be accessed using [CMD/CTRL] + K. When off, channel switcher button is hidden from the sidebar but keyboard shortcut can still be used.",
```

Comment:
From the JIRA ticket, the proposal is to actually use the following text:

```
The channel switcher is shown at the bottom of the sidebar and is used to jump between channels quickly. It can also be accessed using {CMD/CTRL} + K.
```

where {CMD/CTRL} = CMD if on Mac, and CTRL if on Windows